lib/Apache2_4/AuthCookieDBI.pm

```diff
@@ -0,0 +1,225 @@
+#===============================================================================
+#
+# $Id$
+#
+# Apache2_4::AuthCookieDBI
+#
+# A module implementing Apache 2.4.x compatibility for Apache2::AuthCookieDBI
+# group-based authorizations.
+#
+# See end of this file for Copyright notices.
+#
+# Maintainer: Matisse Enzer <matisse@cpan.org> (as of version 2.0)
+#
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+#===============================================================================
+
+package Apache2_4::AuthCookieDBI;
+
+# ABSTRACT: Perl group authorization for Apache 2.4.x
+
+use strict;
```

Comment:
Can we add `use warnings;` I'd really prefer it to be added unless it breaks something we cannot work around.

Reply:
Yes, of course.